server/data/bookAVideoLinkApiClient.test.ts

```diff
@@ -80,4 +84,71 @@ describe('bookAVideoLinkApiClient', () => {
       expect(nock.isDone()).toBe(true)
     })
   })
+
+  describe('get enabled courts', () => {
```

Comment:
This is gonna need to change in the near future is it not?  All courts (including non-enabled).